cypress/integration/iframe.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { IframePage } from "../page/index.ts";
+
+let iframePage: IframePage;
+
+describe("Iframe Interaction", () => {
+  before(() => {
+    iframePage = new IframePage();
+    Cypress.on("uncaught:exception", () => false);
+    iframePage.visit();
+  });
+
+  it("Interacting with the Iframe ", () => {
+    iframePage.getFrame().find("h1").should("have.text", "HTML Tutorial");
```

Comment:
I think it'd be cleaner if you just get the title of the iframe directly, instead of calling `find` again with an h1, since it's not very descriptive and you don't really ever need the whole frame.